<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageGestionMessage</class>
 <widget class="QWidget" name="PageGestionMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QListView" name="intervenantLisetView"/>
   </item>
   <item row="3" column="1">
    <widget class="QListView" name="messageListView"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Intervenants : </string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Reponse</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nom du Ticket</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QListView" name="listView_3"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="ajouterIntervenantButton">
     <property name="text">
      <string>Ajouter un intervenant</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="reglerMessage">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Message </string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="ajouterMessageButton">
     <property name="text">
      <string>Ajouter un message </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
